nitter/docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+services:
+  app_proxy:
+    environment:
+      APP_HOST: $APP_NITTER_IP
+      APP_PORT: 8080
+
+  web:
+    # Official Nim image used as base image for nitter doesn't offer Arm variant so using a custom image
+    # https://github.com/zedeus/nitter/issues/399#issuecomment-997263495
+    image: quay.io/unixfox/nitter:build-20220830@sha256:d93da20aea7dbcf117e0f15bb309f5fc55729b19693d203c1c9708a2fe001582
+    restart: on-failure
+    stop_grace_period: 1m
+    init: true
+    # to pleb to figure out how to run non-root
+    #user: "1000:1000"
+    environment:
+      REDIS_HOST: "redis"
```

Comment:
If we leverage the `entrypoint.sh` script already in the image, then we should only have to populate the env. vars. here for the config. here. Maybe i've missed something as to why this wouldn't work?